src/applications/income-limits/containers/DependentsPage.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import {
+  VaButtonPair,
+  VaNumberInput,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { ROUTES } from '../routes';
```

Comment:
SHOULD

You are getting a bunch of these circular dependency eslint warnings because the routes.jsx imports components which also import from routes.jsx.

I would recommend moving the ROUTES to the 'constants' file and then importing from there rather than from routes.jsx.